includes/class-sensei-assets.php

```diff
@@ -289,4 +289,17 @@ public function get_icon( string $name, string $class_names = '' ) : string {
 
 		return '<svg class="' . esc_attr( $class_names ) . '"><use xlink:href="' . esc_url( $href ) . '"></use></svg>';
 	}
+
+	/**
+	 * Gets image from assets.
+	 *
+	 * @since 3.15.0
```

Comment:
I don't think this is the correct `@since`.

Reply:
[Get image changes](https://github.com/Automattic/sensei/pull/4801/files#diff-10c1e730e0d1f9aea7990d630ddfdbcf57d5750e1739720609c216cc568c7860R294)